### DISCUSSION TITLE
I want to share my mobile network with the router and connect it. The connected device shows zero, and I have enabled DHCP repair, but to no avail.

### DISCUSSION THREAD

**Discussion by york258** [Mygod/VPNHotspot#532] [Q&A]
I want to share my mobile network with the router and connect it. The connected device shows zero, and I have enabled DHCP repair, but to no avail.

**Comment by Mygod** [comment]
Which mode are you using?

**Reply by york258** [reply]
USB connection mode.

**Reply by Mygod** [reply]
Does the connection work if you use USB tethering from system settings without using this app?

**Reply by york258** [reply]
If you don't directly connect to the computer through an application, you can use the network normally. But connecting to the router doesn't work because after connecting to the router, the USB share button is gray.I guess it should be a problem with the router's settings. I chose the DHCP repair option, but it couldn't change anything.
